File: InclusiON.DTOs/Responses/Auth/UserProfileResponse.cs
namespace InclusiON.DTOs.Responses.Auth
{
    public class UserProfileResponse
    {
        public Guid Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Surname { get; set; } = string.Empty;
        public string FullName => $"{Surname}, {Name}".Trim();
        public string Email { get; set; } = string.Empty;
        public string? Phone { get; set; }
        public string Role { get; set; } = string.Empty;
        public bool IsActive { get; set; }
        public int ActiveSessionsCount { get; set; }
        public List<string> Permissions { get; set; } = new();
        public bool EmailConfirmed { get; set; }
        public bool PhoneNumberConfirmed { get; set; }
    }
}

File: src/app/models/responses/user-profile.response.ts
export interface UserProfileResponse {
  id: string;
  name: string;
  surname: string;
  fullName: string;
  email: string;
  phone?: string;
  role: string;
  isActive: boolean;
  activeSessionsCount: number;
  permissions: string[];
  emailConfirmed: boolean;
  phoneNumberConfirmed: boolean;
}
